fakebook/src/app/notifications.service.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable } from '@angular/core';
+import { observable, Observable } from 'rxjs';
+import { HttpClient } from '@angular/common/http';
+import { Notification } from './model/notification';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class NotificationsService {
+
+  constructor(private http: HttpClient) { }
+
+  get Notifications$(): Observable<Notification[]> {
+    return this.getNotificationsObservable();
+  }
+
+  getNotificationsObservable(): Observable<Notification[]> {
+    return undefined as unknown as Observable<Notification[]>;
+  }
```

Comment:
why two methods here?
also `Notifications$` should be `notifications$`
fwiw i thing just `throw`ing something would be simpler but it's fine